pkg/engine/validation.go

```diff
@@ -800,3 +788,29 @@ func matchesException(policyContext *PolicyContext, rule *kyvernov1.Rule) (*kyve
 	}
 	return nil, nil
 }
+
+func shouldRespondException(ctx *PolicyContext, rule *kyvernov1.Rule, log logr.Logger) *response.RuleResponse {
```

Comment:
change return to (bool, *response.RuleResponse)

Reply:
If it returns `true` then we process the response. Otherwise, we expect a nil response.

Reply:
As discussed in channel, comment has been added to make it clear.